### PR TITLE
Add sample choral func for summing values of all children

### PR DESCRIPTION
**Pull request by cpjk**
Later, we can e.g. add this as one of the options for the func of a choral in like a dropdown or something.

### CODE REVIEW

File: sample_chorals/sumChildren.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  /* Sample choral func.
+   * Sum every value of every child and return that
+   * value as the new value of this choral
+   */
+  var children = Object.values(children)
+  var sum = 0;
+  for(var i = 0; i < children.length; i++) {
+    var data = children[i].data;
+    var dataValues = Object.values(data);
+    for (var j = 0; j < dataValues.length; j++) {
```
Comment (andreirtaylor) [inline sample_chorals/sumChildren.js]: So this sums all the values of each member in the children.

Child1
{
   foo: 1,
   bar: 2
}

Child2
{
   foobar: 3,
   barfoo: 4
}

would output 

{
   sum_of_children: 10
}

**Comment by andreirtaylor** [comment]
This looks good to me. If The above comment describes the behaviour.

File: sample_chorals/sumChildren.js (hunk 1 above)
Comment (cpjk) [reply sample_chorals/sumChildren.js]: yes. haven't tested with multiple values for children yet though

Comment (cpjk) [reply sample_chorals/sumChildren.js]: i'll test that actually

**Pull request closed without merging**